### PR TITLE
Make Shortcake work with ACF plugin

### PR DESCRIPTION
**Pull request by tmconnect**
I changed this in my installation to solve issue #90 to make it work with the Advanced Custom Field plugin.

### CODE REVIEW

File: inc/class-shortcode-ui.php [hunk 1]
```diff
@@ -72,7 +72,9 @@ public function get_shortcode( $shortcode_tag ) {
 
 	public function action_admin_enqueue_scripts( $hook ) {
 
-		if ( in_array( $hook, array( 'post.php', 'post-new.php' ) ) ) {
+		global $current_screen;
+
+		if ( in_array( $hook, array( 'post.php', 'post-new.php' ) ) && $current_screen->post_type <> 'acf-field-group') {
```
Comment (danielbachhuber) [inline inc/class-shortcode-ui.php:77]: Better to use `get_current_screen()` than the `$current_screen` global. When you read from global variables, you run the chance of accidentally stomping on them and causing problems later.

**Comment by danielbachhuber** [comment]
I'm not keen on hardcoding a workaround for ACF. Could you describe why the JS error is occurring (screenshots are helpful) so I have a better sense of the root of the problem?

**Comment by danielbachhuber** [comment]
@tmconnect now that #118 is merged, can you test master branch and confirm the fix?

**Comment by tmconnect** [comment]
@danielbachhuber I have tested with ACF 5.1.8 and now it's working and there are no conflicts.

**Comment by danielbachhuber** [comment]
Thanks for the confirmation, @tmconnect

**Pull request closed without merging**